operators/admin/olm-cs-podsched.adoc

```diff
@@ -6,26 +6,48 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its spec:
+When an Operator Lifecycle Manager (OLM) catalog source of source type `grpc` defines a `spec.image`, the Catalog Operator creates a pod that serves the defined image content. By default, this pod defines the following in its specification:
 
 * Only the `kubernetes.io/os=linux` node selector
-* No priority class name
+* A default priority class name
```

Comment:
It's better to point out the default priority class name: `system-cluster-critical`

Reply:
Thanks, @jianzhangbjz . Updated.